tests/arp/test_wr_arp.py

```diff
@@ -57,8 +57,8 @@ def setupFerret(self, duthost, ptfhost):
         ptfhost.copy(src="arp/files/ferret.py", dest="/opt")
 
         result = duthost.shell(
-            cmd='''ip route show proto zebra type unicast |
-            sed -e '/default/d' -ne '/0\//p' |
+            cmd='''ip route show type unicast |
+            sed -e '/proto 186\|proto zebra/!d' -e '/default/d' -ne '/0\//p' |
             head -n 1 |
             sed -ne 's/0\/.*$/1/p'
             '''
```

Comment:
Can you please add a comment what are you trying to get using this commands?

Reply:
Done

Reply:
@tahmed-dev Please don't merge it. I've found a bug in the code. Will fix in an hour and commit.

Reply:
Now it's good